rules/windows/process_creation/proc_creation_win_correlation_msi_privesc.yml

```diff
@@ -0,0 +1,20 @@
+title: Privilege Escalation to System using MSI installers
+id: f1cee536-f60a-4e4f-93a6-20766ef27158
+status: test
+description: Detects privilege escalation using vulnerabilities in MSI installers by correlating the use of msiexec with elevated privileges with spawning of an elevated system shell.
+references:
+    - https://r.sec-consult.com/msi
+    - https://github.com/SigmaHQ/sigma/blob/master/rules-threat-hunting/windows/process_creation/proc_creation_win_susp_elevated_system_shell.yml
+    - https://msrc.microsoft.com/update-guide/en-US/advisory/CVE-2024-38014
+author: Herbert Bärschneider @SEC Consult
+date: 2024-07-15
+tags:
+    - attack.t1068
+    - cve.2024-38014
+correlation:
+    type: temporal
+    rules:
+        - 3e1ce5a2-6c4e-473b-8e55-dffcdd1ffc25 # elevated msiexec spawned
+        - 61065c72-5d7d-44ef-bf41-6a36684b545f # elevated system shell spawned
+    timespan: 30m
```

Comment:
Correlation rules aren't currently supported in the SigmaHQ repo. So'ill delete it for now, until further notice.
Thanks